components/nexusoperations/executors.go

```diff
@@ -747,12 +747,21 @@ func callErrToFailure(callErr error, retryable bool) (*failurepb.Failure, error)
 	if errors.As(callErr, &handlerErr) {
 		failure := &failurepb.Failure{
 			Message: handlerErr.Error(),
-			FailureInfo: &failurepb.Failure_NexusHandlerFailureInfo{
-				NexusHandlerFailureInfo: &failurepb.NexusHandlerFailureInfo{
-					Type: string(handlerErr.Type),
+			FailureInfo: &failurepb.Failure_ApplicationFailureInfo{
+				ApplicationFailureInfo: &failurepb.ApplicationFailureInfo{
+					Type:         "NexusHandlerError",
```

Comment:
Is this the only place? Should it be a constant?

Reply:
Nah, this is all temporary anyways.